reviewing/license-scanner/src/getPkgInfo.ts

```diff
@@ -126,10 +140,18 @@ async function parseLicense (
       } else {
         licenseContents = await readLicenseFileFromCafs(opts.cafsDir, licensePackageFileInfo as PackageFileInfo)
       }
+      const licenseContent = licenseContents?.toString('utf-8')
+      let name = 'Unknown'
+      if (licenseContent) {
+        const match = new RegExp(`\\b(${LICENSE_NAMES.join('|')})\\b`, 'i').exec(licenseContent)
+        if (match) {
+          name = match[0]
+        }
+      }
```

Comment:
I think one license file may contain several licenses.

Reply:
Make sense.

Reply:
Already updated, but I'm not sure if I need to list all the license names according to the information provided by @weyert.